### DISCUSSION TITLE
Coverage factor of statistical uncertainties of scored quantities

### DISCUSSION THREAD

**Discussion by mchamberland** [nrc-cnrc/EGSnrc#1160] [Questions]
What's the coverage factor of the statistical uncertainties of the quantities scored with EGSnrc, e.g., energy deposited? Is it k = 1 or k = 2?  I'm suddenly second-guessing myself and cannot easily find an answer in the documentation.

**Comment by rtownson** [comment]
It's all `k=1` as far as I know.
